api/v1beta1/slicegateway_types.go

```diff
@@ -73,18 +75,26 @@ type SliceGatewayStatus struct {
 	Config SliceGatewayConfig `json:"config,omitempty"`
 	// ConfigUpdatedOn is the time when Config updated from backend
 	ConfigUpdatedOn int64 `json:"configUpdatedOn,omitempty"`
-	// PodName is the name of the gateway pod running in cluster
+	// Deprecated PodName is the name of the gateway pod running in cluster
 	PodName string `json:"podName,omitempty"`
+	// PodNames is the name of the gateway pod running in cluster
```

Comment:
shouldn't it be PodNames is the list of names of gateway pods running in cluster